test/Test01_EquilibSingle.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+import equilipy as eq
+
+#Parse database
+datafile=f'./Database/AlCuMgSi_SK'
+DB=eq.read_dat(datafile+'.dat')
+
+# Parse input data
+NTP = dict({
+    'T':700,
+    'P': 1,
+    'Al':0.060606061,
+    'Cu':0.42424242,
+    'Si':0.515151515})
+
+res=eq.equilib_single(DB,NTP)
+
+
+#Check all stable phases
+print(res.StablePhases['Name'])
```

Comment:
Thanks. I agree. Regression check for values would be more rigorous and useful. However, this will take a while for me to implement it, since i need to get the values from commercial programs.

Currently, what I am checking is if the implemented functions are running smoothly without raising errors as defined in 'Error.py'.